static-site/package.json

```diff
@@ -1,79 +0,0 @@
-{
-  "engines": {
-    "node": "^16",
-    "npm": "^7 || ^8"
-  },
-  "scripts": {
-    "develop": "gatsby develop",
-    "dev": "gatsby develop",
-    "start": "gatsby develop",
-    "serve": "gatsby serve",
-    "build": "gatsby build",
-    "test": "gatsby build",
-    "build:pp": "gatsby build --prefix-paths",
-    "clean": "rm -rf public",
-    "lint": "DEBUG=eslint:cli-engine eslint --ext .js,.jsx .",
-    "lint:md": "remark content/posts/",
-    "heroku-postbuild": "gatsby build"
-  },
-  "dependencies": {
-    "babel-plugin-styled-components": "^1.10.6",
-    "gatsby": "^2.32.13",
-    "gatsby-plugin-catch-links": "^2.1.24",
-    "gatsby-plugin-feed": "^2.3.26",
-    "gatsby-plugin-google-fonts": "^1.0.1",
-    "gatsby-plugin-manifest": "^2.2.37",
-    "gatsby-plugin-nprogress": "^2.1.18",
-    "gatsby-plugin-offline": "^3.0.31",
-    "gatsby-plugin-plausible": "0.0.7",
-    "gatsby-plugin-react-helmet": "^3.1.21",
-    "gatsby-plugin-sharp": "^2.3.13",
-    "gatsby-plugin-sitemap": "^2.2.25",
-    "gatsby-plugin-styled-components": "^3.1.17",
-    "gatsby-plugin-twitter": "^2.1.18",
-    "gatsby-remark-autolink-headers": "^2.1.24",
-    "gatsby-remark-copy-linked-files": "^2.1.36",
-    "gatsby-remark-images": "^3.1.42",
```

Comment:
James mentioned this on a call and indirectly in the PR description, but it's worth noting explicitly that image optimization is removed (previously done by `gatsby-remark-images`). Most notable on pages such as [/blog/2021-07-08-ncov-open-announcement](https://next.nextstrain.org/blog/2021-07-08-ncov-open-announcement) where the size of the image served changes from 32kB to 1.8MB.

It doesn't seem like much of the rest of the site uses large images so this is a minor thing that we can live with.

Reply:
> Most notable on pages such as [/blog/2021-07-08-ncov-open-announcement](https://next.nextstrain.org/blog/2021-07-08-ncov-open-announcement)

I think the blog's the _only_ place -- everywhere else we were (and still are) directly importing / requiring the image so Gatsby wasn't optimising it. I'll double check this and report back.

(P.S. this was included in the commit message for [Move blog to next.js](https://github.com/nextstrain/nextstrain.org/pull/810/commits/36a79a9b656de9be3c0753442363f05c305b043b))

Reply:
Comparing the [140kb ncov splash tile image](https://github.com/nextstrain/nextstrain.org/blob/master/static-site/static/splash_images/ncov.png), both Gatsby + next.js are currently sending the unoptimised image for any screen resolution. This is because we are using `require` rather than just using a URL path for `<img src="...">`, an approach we use throughout the site:

https://github.com/nextstrain/nextstrain.org/blob/f1106c1f438ddcc836b1421d2387d7d0c44e28f8/static-site/src/components/Cards/index.jsx#L22

(I'm not saying this is the best way, but it's the way it's been for a long time and I don't think this PR is the place to change it.)

For the blog we do use a URL path for the `<img>` tags (somewhat indirectly, it's part of the HTML generated from parsing the markdown). So to use the blog post in your example, it links to the 1.7Mb (!) `gisaid_vs_open_map.png`; when viewed on mobile Gatsby serves us a 32kb file whereas on next.js we get the full sized image. 

I think image optimisation is important for us to implement, and the images in the blog are a regression in this PR (although the post you picked is one of the worst!), but I'm hoping it's acceptable to leave this for future work.

Reply:
Thanks for the clarification!

> although the post you picked is one of the worst

Yes, that was specially hand-picked 😅 I think totally fine to leave this for later.